_posts/2021-9-14-gsoc2021-final.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "GSoC 2021 wrap-up"
+---
+
+<p>
+<img
+src="{{ site.baseurl }}{{ site.images }}/mdanalysis-gsoc.png"
+title="Google Summer of Code 2021" alt="Google Summer of Code 2021"
+style="float: right; height: 12em; " />
+</p>
+
+Another year, another successful [Google Summer of Code][gsoc]! We had
+[two fantastic projects][gsoc-projects] this year, including the first trials of our MDAKit model for building an ecosystem of analysis modules. Both projects were a resounding success and represent an exciting new direction for MDAnalysis.  None of this would have been possible without our wonderful students: [Estefania Barreto-Ojeda][project-estefania], and [Orion Cohen][project-orion]. 
+
+You can read more about their projects and packages in their blog posts:
+* Estefania: [Curvature for MDA Universes ]({{ site.baseurl }}{% post_url 2021-09-02-gsoc-final-report-estefania%})
+* Orion: [Solvation Analysis]({{ site.baseurl }}{% post_url 2021-09-02-gsoc-final-report-orion%})
+
+
+Finally, a big thank-you to our GSoC mentors: @fiona-naughton, @lilyminium, @orbeckst,
+@IAlibay, @richardjgowers and @hmacdope. Their dedication, openness and engagement was
+crucial to the success of our students. 
+
+— Org Admins (@richardjgowers, @IAlibay, @orbeckst)
```

Comment:
Let's rewrite
```suggestion
We immensely enjoyed working with our GSoC students and look forward to seeing where they take their projects next.

— Mentors @hmacdope, @lilyminium, @richardjgowers, @fiona-naughton, @orbeckst, @IAlibay
```

Reply:
Alternatively, something along the lines of looking forward to how they continue to contribute to MDAnalysis. But I didn't manage to come up with something good. Even the "seeing where they take their projects next" does not sound quite right to me. Please edit!!!

Reply:
How about "how they continue to contribute to the MDAnalysis community and where they take their science next"
It all feels a bit awkward but I couldn't come up with anything better.